Terratest/vpn/variables.tf

```diff
@@ -0,0 +1,97 @@
+#vpn tunnel configuration:

```

Comment:
while the module itself uses stringified variable names, you can and probably should use objects on the root module level to make the parameters tidier.

Reply:
Defaults or even descriptions are not really useful in context of root module . Also, once again, I do not understand the logic behind adding null defaults.